src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -1501,12 +1591,10 @@ contract AllocationManagerUnitTests_SlashOperator is AllocationManagerUnitTests
         _registerForOperatorSet(defaultOperator, operatorSet);
 
         // Allocate some magnitude, then deallocate some.
-        cheats.startPrank(defaultOperator);
-        allocationManager.modifyAllocations(defaultOperator, allocateParams);
+        _modifyAllocationsWithChecks(defaultOperator, allocateParams);
         cheats.roll(block.number + DEFAULT_OPERATOR_ALLOCATION_DELAY);
-        allocationManager.modifyAllocations(defaultOperator, deallocateParams);
+        _modifyAllocationsWithChecks(defaultOperator, deallocateParams);
```

Comment:
Tests should specifically call the write method, not leave it to an internal function

Reply:
Let's call `modifyAllpocations` in the actual test and then leave checks to an internal function